src/bin/machine-api/cmd_serve.rs

```diff
@@ -125,6 +125,9 @@ pub async fn main(_cli: &Cli, cfg: &Config, bind: &str) -> Result<()> {
             AnyMachine::Moonraker(moonraker) => {
                 spawn_metrics(&mut registry, key, moonraker.get_temperature_sensors()).await?;
             }
+            AnyMachine::BambuX1Carbon(bambu) => {
+                spawn_metrics(&mut registry, key, bambu.get_temperature_sensors()).await?;
```

Comment:
we discovered this is currently broken because this will race the discovery loop and not spawn metrics for printers discovered after the time of this function call. we need to wire this in deeper